### PR TITLE
Tanisha-Model

### PR DESCRIPTION
**Pull request by tanishagarg2**
persisted Book.java and added new class with Robyn's help

### CODE REVIEW

File: src/main/java/com/kyhorne/bookstore/Book.java [hunk 1]
```diff
@@ -23,24 +31,40 @@
...
+    //Cost of the book
+    private Float cost;
+
+    //Quantity of a book in inventory
+    private Integer inventory;
+
+    /**
+     * default constructor
+     */
+    public Book() {
```
Comment (devonverge) [inline src/main/java/com/kyhorne/bookstore/Book.java]: Should this constructor be merged with the other one? Maybe have a field in the other constructor for the amount of this book will be made/added.

File: src/main/java/com/kyhorne/bookstore/Inventory.java [hunk 2]
```diff
@@ -5,46 +5,47 @@
...
      *
      * @param bookRequested
      * @return
      */
-    //TODO: Actually write out the proper code
     public Book retrieveBook(Book bookRequested){
-        return bookRequested;
+        for(Book b: bookInventory){
+            if(b.equals(bookRequested))
+                return b;
```
Comment (tanishagarg2) [inline src/main/java/com/kyhorne/bookstore/Inventory.java]: @kyhorne I wrote this method get the book object from inventory if necessary. I can get rid of it if it seems completely useless

File: src/main/java/com/kyhorne/bookstore/Book.java [hunk 3]
```diff
@@ -23,24 +31,40 @@
     //Publisher of the book
     private String publisher;
 
+    //Cost of the book
+    private Float cost;
+
+    //Quantity of a book in inventory
+    private Integer inventory;
```
Comment (devonverge) [inline src/main/java/com/kyhorne/bookstore/Book.java]: Maybe reword the variable because we have an inventory class so originally looking through code I got confused. Rename to count, amount, etc

Comment (RobynVG) [reply src/main/java/com/kyhorne/bookstore/Book.java]: Hold up I'm in the process of merging

Comment (RobynVG) [reply src/main/java/com/kyhorne/bookstore/Book.java]: Sorry that was a general comment not for this specific one

Comment (devonverge) [reply src/main/java/com/kyhorne/bookstore/Book.java]: @tanishagarg2  @RobynVG  Might want to merge Kyles branch if you havent already into your branches cause it is behind on stuff but has relevant changes.

File: src/main/java/com/kyhorne/bookstore/Book.java (hunk 1 above)
Comment (tanishagarg2) [reply src/main/java/com/kyhorne/bookstore/Book.java]: it is required that all persistent classes have a no-arg constructor according to JDO specification

File: src/main/java/com/kyhorne/bookstore/Book.java (hunk 3 above)
Comment (tanishagarg2) [reply src/main/java/com/kyhorne/bookstore/Book.java]: sounds good

File: src/main/java/com/kyhorne/bookstore/Book.java (hunk 1 above)
Comment (RobynVG) [reply src/main/java/com/kyhorne/bookstore/Book.java]: A default constructor is necessary or it throws an error (unless one is implied by no constructor at all). I'd suggest we get rid of the other one as its not likely to be used
